third_party/glad/README.md

```diff
@@ -0,0 +1,7 @@
+# Introduction
+The `GLAD` library is used for the texture of glfw, which is used to create textures to render RGBA data.
+
+# Notice
+The current `glad` source is a copy from `fuchsia/third_party/glfw/deps` and the first version is copied from https://fuchsia.googlesource.com/third_party/glfw/+/999f3556fdd80983b10051746264489f2cb1ef16/deps.
```

Comment:
I'm not comfortable with checking in a file that we don't apparently understand how to create from the authoritative source, and thus don't know how to update ourselves if we needed to update it.

The [version of this file on master](https://fuchsia.googlesource.com/third_party/glfw/+/refs/heads/master/deps/glad/gl.h) contains the exact details of constructing it from the source, which seems far better. I'm not clear on why we can't do the same thing here instead of relying on this older copy of unclear origin.

Reply:
Sorry, I have always thought that the glad version must be with glfw. After I tested that the glad file from http://glad.sh can still work fine, I proved that I was wrong, glfw and glad are working independently. So, you said that generating `glad` from an authoritative source is correct. I am working on the engine's texture rendering test and will update the PR later.